cbox/__main__.py

```diff
@@ -6,7 +6,8 @@
 import cbox
 from cbox import concurrency
 
-__all__ = ('get_inline_func', 'main', )
+
+__all__ = ('get_inline_func', 'main')
```

Comment:
Please keep the extra comma, apparently it matters in python:

    >>> print ((1,2,3))
    (1, 2, 3)

    >>> print ((1))
    1